lib/restforce/collection.rb

```diff
@@ -31,6 +31,11 @@ def size
     end
     alias_method :length, :size
 
+    # Return whether the collection contains any objects.
```

Comment:
I think I'd rather stick with the minimal interface. Thanks so much for your PR :heart: Please don't be disheartened by me not taking this one - I'm just trying to be opinionated and keep things minimal where we can.